### PR TITLE
Add a test for going from a regular POST into a subflow

### PR DESCRIPTION
**Pull request by spokenbird**
- Adds a test to prevent regression with regards to https://github.com/codeforamerica/form-flow/commit/c05abf334c0e1bc654cd661735d9c9c39c1e9863

### CODE REVIEW

File: src/test/resources/templates/testSubflowLogic/subflowAddItem.html [hunk 1]
```diff
@@ -9,8 +9,8 @@
       <main id="content" role="main" class="form-card spacing-above-35">
         <th:block
             th:replace="~{fragments/cardHeader :: cardHeader(header='Subflow Page', subtext='This is a page within a subflow')}"/>
-        <!-- Test that the currentSubflowItem is passed in by the controller: -->
-        <div th:text="${currentSubflowItem.firstNameSubflow}"></div>
```
Comment (spokenbird) [inline src/test/resources/templates/testSubflowLogic/subflowAddItem.html:13]: I'm not sure what this was doing exactly but firstNameSubflow was null and I couldn't figure out what would have been setting it in the first place.

Comment (bseeger) [reply src/test/resources/templates/testSubflowLogic/subflowAddItem.html:13]: I wonder if that was debug that did get taken out?

Comment (bseeger) [reply src/test/resources/templates/testSubflowLogic/subflowAddItem.html:13]: Should that `div` even be there?  Not exactly sure what it's doing either.

Comment (spokenbird) [reply src/test/resources/templates/testSubflowLogic/subflowAddItem.html:13]: I think you might be right, I'll take it out all together.

**Comment by coltborg** [comment]
@spokenbird, when your back, what's the status of this PR? Is is ready to be reviewed?

File: src/test/resources/templates/testSubflowLogic/subflowAddItem.html (hunk 1 above)
Comment (mpidcock) [reply src/test/resources/templates/testSubflowLogic/subflowAddItem.html:13]: @spokenbird Did you decide to remove this?

**Pull request merged**